protobuf/flow/access/access.proto

```diff
@@ -17,78 +17,101 @@ import "google/protobuf/timestamp.proto";
 service AccessAPI {
 
   // Ping is used to check if the access node is alive and healthy.
-  rpc Ping (PingRequest) returns (PingResponse);
+  rpc Ping(PingRequest) returns (PingResponse);
```

Comment:
can you reset the indentation to what it was. This is the plugin I use to set the indentation automatically -  https://plugins.jetbrains.com/plugin/8277-protobuf-support